llvm/tools/llvm-ir2vec/llvm-ir2vec.cpp

```diff
@@ -182,32 +206,14 @@ class IR2VecTool {
     return Vocab->isValid();
   }
 
-  /// Generate triplets for the module
-  /// Output format: MAX_RELATION=N header followed by relationships
-  void generateTriplets(raw_ostream &OS) const {
-    unsigned MaxRelation = NextRelation; // Track maximum relation ID
-    std::string Relationships;
-    raw_string_ostream RelOS(Relationships);
+  TripletResult getTriplets(const Function &F) const {
```

Comment:
Why change `generateTriplets` to `getTriplets`? AFAIU, it still generates triplets, only that it returns a struct instead of `raw_ostream`.